app/src/test/java/com/example/tasky/feature_agenda/presentation/AgendaViewModelTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.tasky.feature_agenda.presentation
+
+import com.example.tasky.common.domain.TaskyState
+import com.example.tasky.feature_login.domain.repository.UserRemoteRepository
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.impl.annotations.RelaxedMockK
+import junit.framework.Assert.assertEquals
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import org.junit.Test
+
+class AgendaViewModelTest {
+    @RelaxedMockK
+    private lateinit var taskyState: TaskyState
+
+    @RelaxedMockK
+    private lateinit var userRemoteRepository: UserRemoteRepository
```

Comment:
I personally got away from using mocks as much as possible. They're good in two scenarios:
1. You're testing legacy code which isn't written with testability in mind.
2. You're testing code you don't own (e.g. you need to make a library's function return something specific and it's not extendable)

In all other cases, I found fakes/stubs much more reliable and less error prone. I can't count how often my tests broke at some point, not because there was a bug, but because the mock itself broke (for example when the mocked functionality changed/was extended, there were unmocked function calls, etc.). 
Having a fake instead will make you implement new functionality before you can even run your tests and also doesn't rely on that much 3rd party library magic as it's just a plain Kotlin class.